src/main/scala/io/iohk/ethereum/utils/Config.scala

```diff
@@ -38,7 +38,7 @@ object Config {
   object Network {
     private val networkConfig = config.getConfig("network")
 
-    val protocolVersion = networkConfig.getString("protocol-version")
+    val protocolVersion = networkConfig.getInt("protocol-version")
```

Comment:
I misread that, either way I was just curious, we can easily change this at anytime.